railties/lib/rails/generators/rails/plugin/templates/lib/tasks/%namespaced_name%_tasks.rake

```diff
@@ -1,4 +1,4 @@
 # desc "Explaining what the task does"
-# task :<%= name %> do
+# task :<%= underscored_name %> do
```

Comment:
mhh, you are right, namespaced would be really consequent. But imho I think too much of extra code is needed to just throw in comments that 100% reflect how it might/should be used. Now it shows a usage example which is not that 100% pretty but also correct, right?

What do you think?